### PR TITLE
View of CI Builds with re-queue count.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#357

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_007

@amitaibu I've created a view. Please, look at it and tell me if some additional information should be added there. Regarding the issue - I can see that first 2 items has identical problem - user is not a member of the repository.

File: shoov/modules/custom/shoov_ci_build/shoov_ci_build.views_default.inc [hunk 1]
```diff
@@ -371,5 +371,117 @@ function shoov_ci_build_views_default_views() {
...
+  $view->core = 7;
+  $view->api_version = '3.0';
+  $view->disabled = FALSE; /* Edit this to true to make a default view disabled initially */
+
+  /* Display: Master */
+  $handler = $view->new_display('default', 'Master', 'default');
+  $handler->display->display_options['title'] = 'Re-queued items';
+  $handler->display->display_options['use_more_always'] = FALSE;
+  $handler->display->display_options['access']['type'] = 'perm';
+  $handler->display->display_options['access']['perm'] = 'administer nodes';
```
Comment (amitaibu) [inline shoov/modules/custom/shoov_ci_build/shoov_ci_build.views_default.inc]: => administer site configuration

**Comment by amitaibu** [comment]
Can message and uid be links?

**Comment by HelenaEksler** [comment]
> Can message and uid be links?

Sure

**Comment by HelenaEksler** [comment]
@amitaibu Done.
selection_008

**Pull request merged**